superset-frontend/src/components/Label/index.tsx

```diff
@@ -17,149 +17,98 @@
  * under the License.
  */
 import React, { CSSProperties } from 'react';
-import { Label as BootstrapLabel } from 'react-bootstrap';
-import { styled } from '@superset-ui/core';
-import cx from 'classnames';
+import { Tag } from 'src/common/components';
+import { useTheme } from '@superset-ui/core';
 
-export type OnClickHandler = React.MouseEventHandler<BootstrapLabel>;
+export type OnClickHandler = React.MouseEventHandler<HTMLElement>;
+
+export type BsStyle =
```

Comment:
The name BsStyle/bsStyle is a reference to bootstrap. Now that it's out of the picture, we can rename this globaly to something more general, like type/variant/style/etc.